dashboard/config/linux/bits/subsystems.yml

```diff
@@ -26,7 +26,8 @@ config:
  - NUMA_EMU: [x86_64]
  - NUMA_BALANCING: [-arm]
  - NUMA_BALANCING_DEFAULT_ENABLED: [-arm]
- - ADDRESS_MASKING: [x86_64, v6.4]
+ # Starting from v6.13, depends on !CPU_MITIGATIONS.
+ - ADDRESS_MASKING: [x86_64, v6.4, -v6.12]
```

Comment:
cc @ramosian-glider @melver @tarasmadan

Which one of these do we want to test more?
In general this looks concerning -- will we/anybody be able to use LAM at all?